### PR TITLE
Fix Symbol.valueOf method signature

### LINKED ISSUE #13552: Symbol.valueOf method returns a symbol type
http://www.ecma-international.org/ecma-262/6.0/#sec-symbol.prototype.valueof

**TypeScript Version:**  master

**Code**

```diff
interface Symbol {
    /** Returns a string representation of an object. */
    toString(): string;

    /** Returns the primitive value of the specified object. */
-    valueOf(): Object;
+    valueOf(): symbol;
}
```

### PR DESCRIPTION
**Pull request by falsandtru**
Fixes #13552

### CODE REVIEW

File: lib/lib.es6.d.ts [hunk 1]
```diff
@@ -5556,7 +5556,7 @@ interface Symbol {
     toString(): string;
 
     /** Returns the primitive value of the specified object. */
-    valueOf(): Object;
```
Comment (mhegazy) [inline lib/lib.es6.d.ts]: can you revert these changes. all you need is the change to `src/lib/es2015.symbol.d.ts`.

Comment (mhegazy) [inline lib/lib.es6.d.ts]: These files are generated from the build, and are not edited manually.

**Comment by falsandtru** [comment]
done.
